### PR TITLE
Add Coverlet Package to Test Project 

### PR DESCRIPTION
**Pull request by AshantiDDavis**
# 20, Coverlet package missing

## change proposal

- [x] _add coverlet.msbuild package to Xyz.Provider.Tests.csproj
- [ ] _fix ..._

## contributing

this project is maintained under the terms of the [Contribution Guidelines][contribution-guidelines-url].

[contribution-guidelines-url]: https://github.com/revaturexyz/providerxyz/blob/master/.github/CONTRIBUTING.md 'CONTRIBUTION GUIDELINES'

### CODE REVIEW

File: aspnet/Xyz.Provider.Api/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+    "Xyz.Provider.Api": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "api/values",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+    }
+  }
```
Comment (escalonn) [inline aspnet/Xyz.Provider.Api/Properties/launchSettings.json]: these changes do not belong.

File: aspnet/Xyz.Provider.Api/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
+      "Xyz.Provider.Api": {
+        "commandName": "Project",
+        "launchBrowser": true,
+        "launchUrl": "api/values",
+        "applicationUrl": "https://localhost:5001;http://localhost:5000",
+        "environmentVariables": {
+          "ASPNETCORE_ENVIRONMENT": "Development"
+        }
+      }
+    }
```
Comment (escalonn) [inline aspnet/Xyz.Provider.Api/Properties/launchSettings.json]: there is still a big unnecessary whitespace diff (line endings and final newline). it would not cause problems, but we need to learn how to control exactly what is on the branch diff, so i want to see this fully gone.

Comment (escalonn) [reply aspnet/Xyz.Provider.Api/Properties/launchSettings.json]: `git checkout broker aspnet/Xyz.Provider.Api/Properties/launchSettings.json` will put the right version of the file in your working tree.

Comment (AshantiDDavis) [reply aspnet/Xyz.Provider.Api/Properties/launchSettings.json]: I agree that that was a messy solution; I used force-push to undo the change to both files and then only commit the csproj file with the coverlet addition. (the two force pushes are because the first time the packages updated--I repeated the process so that I wouldn't introduce any inconsistencies in package version)

Comment (AshantiDDavis) [reply aspnet/Xyz.Provider.Api/Properties/launchSettings.json]: I didn't know about the "git checkout broker <file>. . ." command I should've checked back here sooner

Comment (escalonn) [reply aspnet/Xyz.Provider.Api/Properties/launchSettings.json]: what package version? do you mean what `dotnet restore` puts into `obj` folder?

Comment (AshantiDDavis) [reply aspnet/Xyz.Provider.Api/Properties/launchSettings.json]: in Visual Studio Community GUI the drop-down to "add a package" is right above the "Update" packages option and I mis-clicked and then canceled the update but not quickly enough because the versions of the NuGet packages were different.
